app/views/users/show.html.erb

```diff
@@ -1,7 +1,7 @@
 <section class="user-section">
   <h2><%= "Name: #{@user.name}" %></h2>
   <% if friends_already(@user) %>
-    <%= button_to 'Remove',friendship_path(finding_friendship_id(@user)), method: :delete %>
+    <%= button_to 'Remove',friendship_path(find_friendship_id(@user)), method: :delete %>
   <% else %>
   <span class="profile-link">
     <%= button_to  'Invite to friendship', friendships_path({:friend_id => @user.id}), method: :post %> 
```

Comment:
That section part of `if..else` statement implies that a connected user will be able to **Invite** himself to friendship